google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -156,8 +164,13 @@ void ModifyTable(google::cloud::bigtable::TableAdmin admin, int argc,
                         google::cloud::bigtable::GcRule::MaxAge(
                             std::chrono::hours(72))))});
 
+    if (!schema) {
+      std::cerr << "ModifyColumnFamilies failed: " << schema.status()
```

Comment:
We recently submitted https://github.com/googleapis/google-cloud-cpp/pull/1957 to the GCS library. The intent was to minimize the error handling noise in the sample code. So rather than a (potentially wrapping) cerr statement followed by a return, we simply `throw` the status wrapped in a runtime_error.

I'm thinking we probably want to do the same in the Bigtable library too, but I'm going to summon @frankyn for his opinion.